<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDatabaseAndTableDialog</class>
 <widget class="QDialog" name="SetDatabaseAndTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetDatabaseAndTableDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>273</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="hostnameLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="portLabel">
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="portLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="databaseNameLabel">
      <property name="text">
       <string>Database Name: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="databaseNameLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tableNameLabel">
      <property name="text">
       <string>Table Name: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tableNameLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>User Name: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string>Host Name/IP address: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>181</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="testConnectionPushButton">
      <property name="text">
       <string>Test Connection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rememberpasswordCheckBox">
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>hostnameLineEdit</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>databaseNameLineEdit</tabstop>
  <tabstop>tableNameLineEdit</tabstop>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
